src/js/init.js

```diff
@@ -1 +1,4 @@
 import './.modernizrrc';
+import cssVars from 'css-vars-ponyfill';
```

Comment:
maybe add modernizr check?

Reply:
should be something like
`
Modernizr.on('customProperties', function( result ) {
  if (result) {
   // the browser has cssVars
  } else {
    // CSSVARS POLIFILL
  }
})
`

Reply:
Modernizer added.